### ISSUE TITLE
BA SRM Torpedo ammo fails to load

### ISSUE THREAD

**Issue by Xeno426** [MegaMek/megameklab#279]
0.45.4
Platform Windows 7 6.1 (x86)
Java version 1.8.0_151

### Description
BA SRM torpedo ammo (at least for Clan BA) can be added to BA in MML, but loading it again or validating it gives an error.
![BA torpedo validation](https://user-images.githubusercontent.com/3220418/54897010-b3ca0b80-4e94-11e9-8679-bc327f42893f.PNG)

**Comment by neoancient** [comment]
I couldn't replicate. Please attached the log and unit files as we requested.

**Comment by Xeno426** [comment]
Ok, I think I traced it back to the Prototype Undine BA. For whatever reason, when created, MML gave it the non-Clan Torpedo ammo, and removing the ammo then adding it in again didn't seem to correct the issue.

[Undine Battle Armor (Prototype)-FC.zip](https://github.com/MegaMek/megameklab/files/3018735/Undine.Battle.Armor.Prototype.-FC.zip)

**Comment by neoancient** [comment]
I'm still not finding a problem. The unit you attached passes validation.

**Comment by Xeno426** [comment]
Because I fixed it. Here's the original, from the current 0.45.4 release.

**Comment by wildj79** [comment]
> On a side note, it would be nice if 7z files were supported.

In MM or on Github? Doesn't make sense in MM because of the cross platform nature of the program. Not that 7z can't be used on other platforms, but it is far from ubiquitous on any system, really. As for Github, you'd have to submit a request to them. The dev's wouldn't have any control over Github's allowed file types for uploads.

**Comment by neoancient** [comment]
It would have been helpful if you had said at the outset that you created the file with an older version of MML and it didn't load on the newer version. It would have taken me less than five minutes to track it down. The problem was a change in the internal name of the Clan torpedo ammo to make it consistent with the other Clan munitions. When I traced it to there I discovered that we were missing SRM torpedo ammo for IS BA.
